doc/man5/config.pod

```diff
@@ -265,8 +265,11 @@ Specifies the pathname of the module (typically a shared library) to load.
 
 =item B<activate>
 
-If present, the module is activated. The value assigned to this name is not
-significant.
+If present and set to one of the values yes, on, true or 1, then the associated
+provider will be activated. Conversely, setting this value to no, off, false, or
+0 will prevent the provider from being activated.  Note that the listed settings
+case insensitive.  Setting activate to any other setting, or omitting a setting
```

Comment:
Sorry for not mentioning this location in the manual pages as well. 
```suggestion
0 will prevent the provider from being activated.  Note that the listed values can be given in lower or UPPER case.  Setting activate to any other setting, or omitting a setting
```

Reply:
(Never mind the change of line wrapping. It was changed by the mobile app)

Reply:
fixed.  Note that UPPER case was converted to uppercase, as doc-nits considers the former to be a violation.